### PR TITLE
Added omit empty annotation on TNS attribute

### PR DESCRIPTION
**Pull request by manjeetsingh02**

### CODE REVIEW

File: soap/client.go [hunk 1]
```diff
@@ -224,7 +224,7 @@ type Envelope struct {
 	XMLName      xml.Name `xml:"SOAP-ENV:Envelope"`
 	EnvelopeAttr string   `xml:"xmlns:SOAP-ENV,attr"`
 	NSAttr       string   `xml:"xmlns:ns,attr"`
-	TNSAttr      string   `xml:"xmlns:tns,attr"`
+	TNSAttr      string   `xml:"xmlns:tns,attr,omitempty"`
```
Comment (8tomat8) [inline soap/client.go]: It will be rewritten on https://github.com/fiorix/wsdl2go/pull/97/files#diff-3bb06cfa05708bb86d2eec5ee5ce82b0R110

Comment (tandr) [reply soap/client.go]: How that would be rewritten, sorry? You are adding `URNAttr`, but not removing `TNSAttr` or modifying its attributes

Comment (8tomat8) [reply soap/client.go]: In the same file. On line 110 you have check
```
if req.TNSAttr == "" {
	req.TNSAttr = req.NSAttr
}
```

**Comment by fiorix** [comment]
👍

**Pull request merged**